go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/kuadrant/authorino-operator v0.11.1
 	github.com/kuadrant/dns-operator v0.0.0-20241018131559-f2ce8b6aaaef
 	github.com/kuadrant/limitador-operator v0.9.0
-	github.com/kuadrant/policy-machinery v0.5.0
+	github.com/kuadrant/policy-machinery v0.5.1-0.20241020183426-2dc665c31409
```

Comment:
Merge this PR only after main of policy-machinery is tagged and kuadrant operator is updated in a separate PR that @KevFan is creating.

Reply:
https://github.com/Kuadrant/kuadrant-operator/pull/950